src/99-appendices/08-coordinate-systems.md

```diff
@@ -1,5 +1,49 @@
 # Appendix VIII: preferred names of Coordinate systems
 
+## Definitions
+
+### Coordinate System
+
+A coordinate system specifies the information necessary to interpret the
+coordinates associated with the data. A coordinate system can be described by
+its topology (cartesian, spherical, polar, etc.,.), dimensionality, and units of
+measurement along each dimension. Every device that makes spatial measurements
+or has sensors described in a space has a coordinate system (e.g., an MR
+scanner, an MEG scanner, an EEG cap, Polhemus digitizer). The data output by
```

Comment:
an EEG cap is not a spatial measurement device and has no intrinsic coordinates. It is the Polhemus that measures coordinates of electrodes.

Reply:
Could you suggest a better wording (roping in @satra for additional comments).

Reply:
@robertoostenveld - better wording would be great. but as far as i know the cap is used by people to refer to coordinates of channels, without doing any digitization - https://en.wikipedia.org/wiki/10%E2%80%9320_system_(EEG)

Reply:
@satra you are confusing "electrode placement system" (the design of the schema) with "electrode cap" (the flexible fabric). The 10-20 system does not depend on a cap, but does require measurements. To make electrode application faster, researchers often skip the detailed measurement but use a cap (with only a few measurements) instead. See also 
https://doi.org/10.1016/S1388-2457(00)00527-7

Reply:
From my current reading of the contents of this PR the section "Definitions" in Appendix VIII that this discussion revolves around is not crucial for clarifying the use of any of the data types described elsewhere in the PR. Perhaps we could move it to a new PR and work on it on an independent schedule?